tensorboard/http_api.md

```diff
@@ -35,13 +35,30 @@ The `logdir` argument is the path of the directory that contains events files.
 
 ## `data/plugins_listing`
 
-Returns a dict mapping from plugin name to a boolean indicating whether
+Returns a dict mapping from plugin name to an object that describes a plugin.
```

Comment:
Previously, dashboard order was determined by plugin registration order,
which was in turn hard-coded in `default-plugins.html`. But now there is
nothing to determine plugin registration order. Should this be an array
of plugin objects instead?

Reply:
Discussed a bit offline:
1. the order of plugins in TensorBoard UI is a UI consideration and plugins_listing probably should not care about that (sorting should be done on the frontend)
2. currently, the order in which plugins are listed is dictated by order of `registerDashboard` calls and it is defined in https://github.com/tensorflow/tensorboard/blob/master/tensorboard/components/tf_tensorboard/default-plugins.html. 
3. For dynamic plugins, the frontend entry point will not appear in aforementioned default-plugins.html. In future FE changes, we will _likely_ to introduce some business logic around how plugins are sorted in UI: plugins listed on the default-plugins will appear first then dynamic plugins will be alphabetically sorted.